rllib/algorithms/algorithm.py

```diff
@@ -322,8 +322,10 @@ def __init__(
             **kwargs: Arguments passed to the Trainable base class.
         """
 
-        # Resolve possible dict into an AlgorithmConfig object.
-        # TODO: In the future, only support AlgorithmConfig objects here.
+        # Resolve possible dict into an AlgorithmConfig object as well as
```

Comment:
According to the type descriptors in the function signature, we don't accept dicts here anymore!

Reply:
(If we still do, we should send a deprecation warning?)

Reply:
True! Good point.

Reply:
I still want to support it for a while, but yes, we should warn.